pwa/package.json

```diff
@@ -32,7 +32,8 @@
     "react-flash-message": "^1.0.8",
     "react-helmet": "^6.1.0",
     "react-movable": "^3.0.2",
-    "typescript": "^4.4.4"
+    "typescript": "^4.4.4",
+    "validator": "^13.7.0"
```

Comment:
I think we might not need this `validator` package, since we're only using it to valide if an input holds an `url`.

Input fields have a `type=url` option, [more documentation here](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/url).

Maybe see if you can manage to validate the field without the package.